tests/core/server/test_rate_limits.py

```diff
@@ -332,3 +332,26 @@ async def test_different_versions(self, node_with_params, node_with_params_b, se
         test_different_versions_results.append(total_tx_msg_count)
         if len(test_different_versions_results) >= 4:
             assert len(set(test_different_versions_results)) >= 2
+
+    @pytest.mark.asyncio
```

Comment:
it doesn't look like this needs to be marked `asyncio`, nor be a method. but I suppose that's a nit-pick